wremnants/syst_tools.py

```diff
@@ -388,7 +400,7 @@ def add_qcdScale_hist(results, df, axes, cols, base_name="nominal", addhelicity=
 def add_qcdScaleByHelicityUnc_hist(results, df, helper, axes, cols, base_name="nominal", addhelicity=False):
     name = Datagroups.histName(base_name, syst="qcdScaleByHelicity")
     if "helicityWeight_tensor" not in df.GetColumnNames():
-        df = df.Define("helicityWeight_tensor", helper, ["massVgen", "absYVgen", "ptVgen", "chargeVgen", "csSineCosThetaPhi", "scaleWeights_tensor", "nominal_weight"])
+        df = df.Define("helicityWeight_tensor", helper, ["massVgen", "absYVgen", "ptVgen", "chargeVgen", "csSineCosThetaPhi", "nominal_weight"])
     if addhelicity:
         qcdbyHelicity, qcdbyHelicity_axes = make_qcdscale_helper_helicity(helper.tensor_axes)
         df = df.Define('scaleWeights_tensor_wnom_helicity', qcdbyHelicity, ['helicityWeight_tensor', 'helWeight_tensor'])
```

Comment:
Did you manage to see if it still works when addhelicity=True ? In fact I am not sure where or whether we use this flag anymore.

Reply:
Indeed this is much less flexible than before and probably some of the "extra" functionality doesn't work anymore.  We can clean this up later though.